app/containers/Toast.tsx

```diff
@@ -25,50 +25,32 @@ interface IToastProps {
 	theme?: TSupportedThemes;
 }
 
-class Toast extends React.Component<IToastProps, any> {
-	private listener?: Function;
-
-	private toast: EasyToast | null | undefined;
-
-	componentDidMount() {
-		this.listener = EventEmitter.addEventListener(LISTENER, this.showToast);
-	}
-
-	shouldComponentUpdate(nextProps: any) {
-		const { theme } = this.props;
-		if (nextProps.theme !== theme) {
-			return true;
-		}
-		return false;
-	}
-
-	componentWillUnmount() {
-		if (this.listener) {
-			EventEmitter.removeListener(LISTENER, this.listener);
-		}
-	}
-
-	getToastRef = (toast: EasyToast | null) => (this.toast = toast);
-
-	showToast = ({ message }: { message: string }) => {
-		if (this.toast && this.toast.show) {
-			this.toast.show(message, 1000);
+const Toast = ({ theme }: IToastProps) => {
+	let listner: Function;
+	let toast: EasyToast | null | undefined;
+	useEffect(() => {
+		listner = EventEmitter.addEventListener(LISTENER, showToast);
+		return () => {
+			EventEmitter.removeListener(LISTENER, listner);
+		};
+	});
+	const getToastRef = (newToast: EasyToast | null) => (toast = newToast);
+
+	const showToast = ({ message }: { message: string }) => {
+		if (toast && toast.show) {
+			toast.show(message, 1000);
 		}
 	};
-
-	render() {
-		const { theme } = this.props;
-		return (
-			<EasyToast
-				ref={this.getToastRef}
-				// @ts-ignore
-				position='center'
-				style={[styles.toast, { backgroundColor: themes[theme!].toastBackground }]}
-				textStyle={[styles.text, { color: themes[theme!].buttonText }]}
-				opacity={0.9}
-			/>
-		);
-	}
-}
+	return (
+		<EasyToast
+			ref={getToastRef}
+			// @ts-ignore
+			position='center'
+			style={[styles.toast, { backgroundColor: themes[theme!].toastBackground }]}
```

Comment:
By using `colors` from the `useTheme` hook, you can change `themes[theme!].toastBackground` with `colors.toastBackground`